server/index.js

```diff
@@ -1,19 +1,19 @@
-// LIBRARY IMPORTS
+// imports
 require('dotenv').config();
-
 const express = require('express');
 const morgan = require('morgan');
 const cors = require('cors');
 const path = require('path');
 
 const app = express();
 
-// ADD APP-WIDE MIDDLEWARE
+// middleware
 app.use(morgan('dev'));
 app.use(cors());
 app.use(express.json());
 
 app.use(express.static(path.join(__dirname, '../test')));
+app.use('/products', router);
```

Comment:
did you forget to require router? or am I not seeing it?

Reply:
yes i did! good catch!